docs/eventing/brokers/create-broker.md

```diff
@@ -0,0 +1,116 @@
+
+        ```bash
+        kn broker create <broker-name> -n <namespace>
+        ```
+ 
+        This will create a new Broker of your default broker class and default broker configuration (both defined in the `config-br-defaults` ConfigMap).
+ 
+        !!! note
+            If you choose not to specify a namespace, the broker will be created in the current namespace.
+ 
+        !!! note
+            If you have multiple Broker classes installed in your cluster, you can specify the broker class via the `--class` parameter, e.g.:
+ 
+            ```bash
+            kn broker create <broker-name> -n <namespace> --class MTChannelBasedBroker
+            ```
+
+    1. Optional: Verify that the broker was created by listing existing brokers:
+
+        ```bash
+        kn broker list
+        ```
+
+    1. Optional: You can also verify the broker exists by describing the broker you have created:
+
+        ```bash
+        kn broker describe <broker-name>
+        ```
+
+=== "kubectl"
+
+    The YAML in the following example creates a broker named `default`.
+
+    1. Create a Broker by creating a YAML file using the following template:
+
+        ```yaml
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        metadata:
+          name: <broker-name>
+          namespace: <namespace>
+        ```
+ 
+        This will create a new Broker of your default broker class and default broker configuration (both defined in the `config-br-defaults` ConfigMap).
+
+    1. Apply the YAML file:
+
+        ```bash
+        kubectl apply -f <filename>.yaml
+        ```
+        Where `<filename>` is the name of the file you created in the previous step.
+
+    1. Optional: Verify that the broker is working correctly:
+
+        ```bash
+        kubectl -n <namespace> get broker <broker-name>
+        ```
+ 
+        This shows information about your broker. If the broker is working correctly, it shows a `READY` status of `True`:
+ 
+        ```bash
+        NAME      READY   REASON   URL                                                                        AGE
+        default   True             http://broker-ingress.knative-eventing.svc.cluster.local/default/default   1m
+        ```
+ 
+        If the `READY` status is `False`, wait a few moments and then run the command again.
+
+## Broker class options
+
+When a Broker is created without a specified `eventing.knative.dev/broker.class` annotation, the default `MTChannelBasedBroker` Broker class is used, as specified by default in the `config-br-defaults` ConfigMap. 
+
+In case you have multiple Broker classes installed in your cluster and want to use a non-default Broker class for a Broker, you can modify the `eventing.knative.dev/broker.class` annotation and `spec.config` for the Broker object.
+
+1. Modify the `eventing.knative.dev/broker.class` annotation. Replace `MTChannelBasedBroker` with the class type you want to use:
```

Comment:
The broker class is immutable, I fear "can modify" and "replace" can be read as "update the object in your cluster", can we clarify that?

Reply:
I see... Will update it